src/backend/optimizer/plan/orca.c

```diff
@@ -55,21 +55,21 @@ log_optimizer(PlannedStmt *plan, bool fUnexpectedFailure)
 	/* optimizer failed to produce a plan, log failure */
 	if (OPTIMIZER_ALL_FAIL == optimizer_log_failure)
 	{
-		elog(LOG, "Planner produced plan :%d", fUnexpectedFailure);
+		elog(LOG, "Pivotal Optimizer produced plan :%d", fUnexpectedFailure);
```

Comment:
can we be consistent with https://github.com/greenplum-db/gpdb/pull/7511 and change this to the following

```suggestion
		elog(LOG, "Pivotal Optimizer (GPORCA) produced plan :%d", fUnexpectedFailure);
```

Reply:
Done.